xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.h

```diff
@@ -122,10 +122,10 @@ class CActiveAESink : private CThread
   CEvent *m_inMsgEvent;
   int m_state;
   bool m_bStateMachineSelfTrigger;
-  std::chrono::milliseconds m_extTimeout;
-  std::chrono::milliseconds m_silenceTimeOut;
+  std::chrono::milliseconds m_extTimeout{static_cast<std::chrono::milliseconds>(0)};
```

Comment:
The static_cast shouldn't be needed

Reply:
Is to avoid include `using namespace std::chrono_literals` in the header which is included also in many .cpp files

Reply:
It is not needed this time. Should have been done when the chrono stuff was pushed, which introduced the issue. I would suggest to handle the "Always" case for now explicitely as this restores original functionality. Rewriting the settings handlers will be too intrusive anyways for a v20 fix, which needs to happen, right?

Reply:
> It is not needed this time. Should have been done when the chrono stuff was pushed, which introduced the issue.
+1

Reply:
> Is to avoid include `using namespace std::chrono_literals` in the header which is included also in many .cpp files

You don't need to use the std::chrono_literals namespace

Reply:
ref: https://godbolt.org/z/xoaYjzcE3

Reply:
Done, removed `static_cast`